core/src/main/resources/jenkins/tasks/filters/impl/RetainVariablesLocalRule/config_fr.properties

```
@@ -0,0 +1,26 @@
+# The MIT License
+#
+# Copyright (c) 2020, CloudBees, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+variablesLabel=Variables d''environnement à garder
+variablesDescription=Toutes les autres variables d''environnement fournies par Jenkins seront supprimées. \
+  Les variables d''environnement définies hors de Jenkins seront réinitialisées à leur valeur par défaut ou supprimées, en fonction de l''option "Gestion des variables d''environnement", sauf si spécifié ici.
+retainCharacteristicEnvVarsLabel=Garde les variables d''environnement caractéristiques
```

```suggestion
retainCharacteristicEnvVarsLabel=Garde les variables d''environnement spécifiques
```
Make more sense in French like that IMO (optional)